code-studio/src/js/levels/levelGroup.js

```diff
@@ -68,7 +68,11 @@ window.initLevelGroup = function (
 
   var lastResponse = window.getResult().response;
 
-  window.levelGroup.answerChangedFn = function (levelId) {
+  window.levelGroup.answerChangedFn = function (levelId, commit) {
+    if (!commit) {
```

Comment:
What does `commit` mean here?  It's not obvious from the parameter name that it's boolean, and it's kind of an overloaded term (is this a git commit, or the student committing to a particular answer?)  Maybe `commitToAnswer` would be a better name?